src/Text/Blaze/Svg/Internal.hs

```diff
@@ -194,8 +194,8 @@ aa
   => a -- ^ Radius in the x-direction
   -> a -- ^ Radius in the y-direction
   -> a -- ^ The rotation of the arc's x-axis compared to the normal x-axis
-  -> a -- ^ Draw the smaller or bigger arc satisfying the start point
-  -> a -- ^ To mirror or not
+  -> Bool -- ^ Draw the smaller or bigger arc satisfying the start point
```

Comment:
Could you clarify in the documentation which of (True, False) corresponds to (smaller, bigger)?